packages/core/less/message-dialog.less

```diff
@@ -0,0 +1,26 @@
+.message-dialog-pf-body {
+  display: flex;
+  color: #363636;
+}
+
+.message-dialog-pf-icon {
+  font-size: 30px;
+  margin-right: 15px;
+  display: flex;
+  flex-direction: column;
```

Comment:
Can we remove the flex styles here, and then remove the `<div>` and move the class to the `<span>` like below:
```
<span aria-hidden="true" class="pficon pficon-info message-dialog-pf-icon"></span>
```

Reply:
The CSS is being reworked into patternfly/patternfly, see https://github.com/patternfly/patternfly/pull/1074

Reply:
Oh, no worries. I was asking @matthewcarleton yesterday about some of the styles, and he mentioned the same thing. We were just now chatting and comparing notes. These last comments were ones that I had made while we were reviewing the styles together, and I wanted to go ahead and capture them even though we're making these changes in pf core. I think he's going to follow up with you in the pf core PR about this.